File: Archiver/frmAgreement.cs
using System;
using MODI;

namespace EcmArchiver
{
    public partial class frmAgreement
    {
        public frmAgreement()
        {
            InitializeComponent();
            _ckAgree.Name = "ckAgree";
            _ckDisagree.Name = "ckDisagree";
            _btnProcess.Name = "btnProcess";
        }

        private void btnProcess_Click(object sender, EventArgs e)
        {
            Dispose();
        }

        private void ckAgree_CheckedChanged(object sender, EventArgs e)
        {
            if (ckAgree.Checked == true)
            {
                modGlobals.gLegalAgree = true;
                ckDisagree.Checked = false;
            }
        }

        private void ckDisagree_CheckedChanged(object sender, EventArgs e)
        {
            if (ckDisagree.Checked == true)
            {
                modGlobals.gLegalAgree = false;
                ckAgree.Checked = false;
            }
        }

        private void frmAgreement_Load(object sender, EventArgs e)
        {
            ckDisagree.Checked = true;
        }
    }
}

File: Archiver/frmAgreement.vb
Public Class frmAgreement

    Private Sub btnProcess_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnProcess.Click
        Me.Dispose()
    End Sub

    Private Sub ckAgree_CheckedChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ckAgree.CheckedChanged
        If ckAgree.Checked = True Then
            gLegalAgree = True            
            Me.ckDisagree.Checked = False
        End If
    End Sub

    Private Sub ckDisagree_CheckedChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ckDisagree.CheckedChanged
        If ckDisagree.Checked = True Then
            gLegalAgree = False
            ckAgree.Checked = False
        End If
    End Sub

    Private Sub frmAgreement_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        ckDisagree.Checked = True
    End Sub
End Class
